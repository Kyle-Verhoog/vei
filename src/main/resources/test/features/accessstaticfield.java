public class Accessstaticfield {
  public Accessstaticfield() {}
  public int m() {
    return java.util.Calendar.MONDAY;
  }
}