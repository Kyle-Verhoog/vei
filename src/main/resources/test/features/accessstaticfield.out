PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
) ┃ )
{ ┃ {
RETURN ┃ return
IDENTIFIER ┃ java
. ┃ .
IDENTIFIER ┃ util
. ┃ .
IDENTIFIER ┃ Calendar
. ┃ .
IDENTIFIER ┃ MONDAY
; ┃ ;
} ┃ }
} ┃ }